spice/wrapped_clients.py

```diff
@@ -203,14 +272,41 @@ def catch_and_convert_errors(self):
         except anthropic.AuthenticationError as e:
             raise AuthenticationError(f"Anthropic Error: {e.message}") from e
 
+    # Anthropic doesn't give us a way to count tokens, so we just use OpenAI's token counting functions and multiply by a pre-determined multiplier
+    class _FakeWrappedOpenAIClient(WrappedOpenAIClient):
+        def __init__(self):
+            pass
+
+    _fake_openai_client = _FakeWrappedOpenAIClient()
+    # TODO: Figure out a good starting multiplier
+    _anthropic_token_multiplier = 1
```

Comment:
I got values ranging from 1.15 to 1.3ish. it'll differ by content type though. I suggest we use 1.25